ROADMAP.md

```diff
@@ -0,0 +1,10 @@
+# Chaosd Roadmap
+
+This document defines the roadmap for Chaosd development.
+
+## v2.0
+
+- [ ] Support HTTP attack to simulate HTTP faults, such as abort connection, delay, etc.
+- [ ] Support IO attack to simulate file system faults, such as IO delay and read/write errors.
+- [ ] Support workflow to manage a group of chaos experiments.
+- [ ] Support use Dashboard to manage chaos experiments.
```

Comment:
I think dashboard may be hard to be supported in v2.0.

Reply:
done